src/demod.c

```diff
@@ -202,16 +203,24 @@ demod_thr_qpsk(void* x)
 				/* Costas loop frequency/phase tuning */
 				cur = costas_mix(self->cst, cur);
 				costas_correct_phase(self->cst, costas_delta(cur, cur));
+				
+				if(self->cst->locked) {
+					self->writeStarted = true;
```

Comment:
So you've set a variable, but where should it be used?

Reply:
Sorry, you are totally correct. I wrote this code a couple of month ago, and I had two versions, one that used args->self->writeStarted to mark start of saving data, and one where self->cs->locked is used for this purpose. I am going to fix the code.